.github/ci/packages-jammy.apt

```diff
@@ -0,0 +1 @@
+python3-distutils
```

Comment:
Are you opposed to removing `python3-distutils` from any of the `.apt` files? I think it would be fine since Jammy has a newer version of cmake.

Reply:
I think it would be fine to remove for Harmonic since 22.04 has cmake 3.22.1